plugins/modules/digital_ocean_droplet.py

```diff
@@ -315,22 +330,23 @@ def main():
             oauth_token=dict(
                 aliases=['API_TOKEN'],
                 no_log=True,
-                fallback=(env_fallback, ['DO_API_TOKEN', 'DO_API_KEY', 'DO_OAUTH_TOKEN'])
+                fallback=(env_fallback, ['DO_API_TOKEN', 'DO_API_KEY', 'DO_OAUTH_TOKEN']),
+                required=True,
```

Comment:
This change definitely needs a changelog fragment, it's a potentially breaking change (except in case the module didn't work before when it wasn't specified, then it's a bugfix).

Reply:
Looks like we don't need to require it since the module fails gracefully if it's undefined:

```
❯ env | grep -E 'DO_|API_'
❯ ansible-playbook -i localhost, -c local droplet.yml -v 2>/dev/null | grep msg:
  msg: Failed to login using API token, please verify validity of API token.
```

Reply:
Looks like we don't need to require it since the module fails gracefully if it's undefined:

```
❯ env | grep -E 'DO_|API_'
❯ ansible-playbook -i localhost, -c local droplet.yml -v 2>/dev/null | grep msg:
  msg: Failed to login using API token, please verify validity of API token.
```

Reply:
In that case it's not a breaking change :) Requiring it is still a good idea, so the error handling is more uniform.